nixos/modules/services/monitoring/pgscv.nix

```
@@ -0,0 +1,67 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (lib)
+    mkEnableOption
+    mkIf
+    mkOption
+    types
+    ;
+
+  cfg = config.services.pgscv;
+
+  settingsFormat = pkgs.formats.yaml { };
+  configFile = settingsFormat.generate "config.yaml" cfg.settings;
+in
+{
+  options.services.pgscv = {
+    enable = mkEnableOption "pgSCV, a PostgreSQL ecosystem metrics collector";
```

```suggestion
    enable = mkEnableOption "pgSCV, a PostgreSQL ecosystem metrics collector";
    
    package = mkPackageOption pkgs "pgscv" { };
```

It's best practice to expose the package used to allow for a clear approach for overwrites w/o overlays.

---

Done.